src/pages/workspace/accounting/xero/XeroImportPage.tsx

```diff
@@ -43,7 +43,9 @@ function XeroImportPage({policy}: WithPolicyProps) {
             },
             {
                 description: translate('workspace.xero.customers'),
-                action: () => {},
+                action: () => {
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_XERO_CUSTOMER.getRoute(policyID));
+                },
                 hasError: !!policy?.errors?.importCustomers,
                 title: importCustomers ? translate('workspace.accounting.importedAsTags') : '',
```

Comment:
@hungvu193 bug: the imports page should not show imported when this setting is off.

https://github.com/Expensify/App/assets/29673073/ca89f9ae-d36c-4e91-9f58-73b68949747e

Reply:
Yeah, I saw it. The value of Import page depends on `importCustomers` while I'm updating `syncCustomers`, I raised a question on slack.